theatre/serializers.py

```diff
@@ -0,0 +1,148 @@
+from django.db import transaction
+from rest_framework import serializers
+
+from theatre.models import (
+    Genre,
+    Actor,
+    TheatreHall,
+    Play,
+    Performance,
+    Ticket,
+    Reservation,
+)
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ("id", "name")
+
+
+class GenreDetailSerializer(GenreSerializer):
+    pass
+
+    class Meta:
+        model = Genre
+        fields = ("id", "name")
```

Comment:
As for me this Serializers are identical. I think if you redefine class Meta in Detail serializer you don't need to have 'pass' there. I think it's even better to leave pass here but delete class Meta because you inherit it from 'GenreSerializer'